[2024-12-16 22:44:18,225: INFO: main: welcome to our custom log]
[2024-12-17 01:05:07,721: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-17 01:05:07,726: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 01:05:07,728: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 01:05:07,730: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 01:05:07,731: INFO: common: created directory at: artifacts]
[2024-12-17 01:05:07,733: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-17 01:05:09,151: INFO: data_ingestion: artifacts/data_ingestion/data.zip download! with following info: 
Connection: close
Content-Length: 23329
Cache-Control: max-age=300
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Content-Type: application/zip
ETag: "c69888a4ae59bc5a893392785a938ccd4937981c06ba8a9d6a21aa52b4ab5b6e"
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
X-GitHub-Request-Id: 2B66:21564D:1DE241B:1EB8D51:6760B224
Accept-Ranges: bytes
Date: Mon, 16 Dec 2024 23:05:09 GMT
Via: 1.1 varnish
X-Served-By: cache-fra-eddf8230024-FRA
X-Cache: MISS
X-Cache-Hits: 0
X-Timer: S1734390309.082880,VS0,VE223
Vary: Authorization,Accept-Encoding,Origin
Access-Control-Allow-Origin: *
Cross-Origin-Resource-Policy: cross-origin
X-Fastly-Request-ID: 023d50417bef8ddf8698e22c76329d66542d6778
Expires: Mon, 16 Dec 2024 23:10:09 GMT
Source-Age: 0

]
[2024-12-17 01:05:09,181: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-17 02:27:49,235: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-17 02:27:49,241: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 02:27:49,243: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 02:27:49,245: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 02:27:49,246: INFO: common: created directory at: artifacts]
[2024-12-17 02:27:49,249: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-17 02:27:49,250: INFO: data_ingestion: File already exists]
[2024-12-17 02:27:49,296: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-17 21:38:42,094: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-17 21:38:42,102: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 21:38:42,110: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 21:38:42,117: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 21:38:42,123: INFO: common: created directory at: artifacts]
[2024-12-17 21:38:42,124: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-17 21:38:42,131: INFO: data_ingestion: File already exists]
[2024-12-17 21:38:42,155: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-17 21:41:49,309: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-17 21:41:49,313: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 21:41:49,314: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 21:41:49,317: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 21:41:49,318: INFO: common: created directory at: artifacts]
[2024-12-17 21:41:49,319: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-17 21:41:49,319: INFO: data_ingestion: File already exists]
[2024-12-17 21:41:49,329: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-17 21:41:49,329: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-17 21:41:49,330: ERROR: main: name 'DataValidationTrainingPipeline' is not defined]
Traceback (most recent call last):
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\main.py", line 16, in <module>
    obj = DataValidationTrainingPipeline()
NameError: name 'DataValidationTrainingPipeline' is not defined
[2024-12-17 21:43:11,864: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-17 21:43:11,870: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 21:43:11,873: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 21:43:11,877: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 21:43:11,882: INFO: common: created directory at: artifacts]
[2024-12-17 21:43:11,884: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-17 21:43:11,885: INFO: data_ingestion: File already exists]
[2024-12-17 21:43:11,904: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-17 21:43:11,907: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-12-17 21:43:11,922: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 21:43:11,923: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 21:43:11,935: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 21:43:11,943: INFO: common: created directory at: artifacts]
[2024-12-17 21:43:11,945: ERROR: main: "'ConfigBox' object has no attribute 'data'"]
Traceback (most recent call last):
  File "box\\box.py", line 593, in box.box.Box.__getitem__
KeyError: 'data'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 632, in box.box.Box.__getattr__
  File "box\\box.py", line 620, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'data'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 634, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'data'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\config_box.py", line 29, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 648, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'data'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 593, in box.box.Box.__getitem__
KeyError: 'data'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "box\\box.py", line 632, in box.box.Box.__getattr__
  File "box\\box.py", line 620, in box.box.Box.__getitem__
box.exceptions.BoxKeyError: "'data'"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "box\\box.py", line 634, in box.box.Box.__getattr__
AttributeError: 'ConfigBox' object has no attribute 'data'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\main.py", line 20, in <module>
    data_ingestion.initiate_data_validation()
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\src\datascience\pipeline\data_validation_pipeline.py", line 12, in initiate_data_validation
    data_validation_config = config.get_data_validation_config()
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\src\datascience\config\configuration.py", line 30, in get_data_validation_config
    config = self.config.data.data_validation
  File "box\\config_box.py", line 31, in box.config_box.ConfigBox.__getattr__
  File "box\\box.py", line 648, in box.box.Box.__getattr__
box.exceptions.BoxKeyError: "'ConfigBox' object has no attribute 'data'"
[2024-12-17 21:44:17,979: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-17 21:44:17,983: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 21:44:17,985: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 21:44:17,989: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 21:44:17,990: INFO: common: created directory at: artifacts]
[2024-12-17 21:44:17,992: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-17 21:44:17,994: INFO: data_ingestion: File already exists]
[2024-12-17 21:44:18,007: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-17 21:44:18,011: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-12-17 21:44:18,015: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 21:44:18,019: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 21:44:18,029: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 21:44:18,034: INFO: common: created directory at: artifacts]
[2024-12-17 21:44:18,036: INFO: common: created directory at: artifacts/data_validation]
[2024-12-17 21:44:18,037: ERROR: main: name 'DataValiadtion' is not defined]
Traceback (most recent call last):
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\main.py", line 20, in <module>
    data_ingestion.initiate_data_validation()
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\src\datascience\pipeline\data_validation_pipeline.py", line 13, in initiate_data_validation
    data_validation = DataValiadtion(config = data_validation_config)
NameError: name 'DataValiadtion' is not defined
[2024-12-17 21:45:59,918: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-17 21:45:59,922: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 21:45:59,931: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 21:45:59,948: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 21:45:59,950: INFO: common: created directory at: artifacts]
[2024-12-17 21:45:59,951: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-17 21:45:59,952: INFO: data_ingestion: File already exists]
[2024-12-17 21:45:59,967: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-17 21:45:59,970: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-12-17 21:45:59,979: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 21:45:59,981: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 21:45:59,985: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 21:45:59,987: INFO: common: created directory at: artifacts]
[2024-12-17 21:45:59,990: INFO: common: created directory at: artifacts/data_validation]
[2024-12-17 21:46:00,080: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-12-17 22:23:18,720: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-17 22:23:18,727: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 22:23:18,731: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 22:23:18,734: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 22:23:18,736: INFO: common: created directory at: artifacts]
[2024-12-17 22:23:18,737: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-17 22:23:18,737: INFO: data_ingestion: File already exists]
[2024-12-17 22:23:18,755: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-17 22:23:18,758: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-12-17 22:23:18,767: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 22:23:18,769: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 22:23:18,774: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 22:23:18,778: INFO: common: created directory at: artifacts]
[2024-12-17 22:23:18,779: INFO: common: created directory at: artifacts/data_validation]
[2024-12-17 22:23:18,854: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-12-17 22:23:18,855: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-12-17 22:23:18,856: ERROR: main: DataTransformation.__init__() missing 1 required positional argument: 'config']
Traceback (most recent call last):
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\main.py", line 30, in <module>
    data_transformation = DataTransformation()
TypeError: DataTransformation.__init__() missing 1 required positional argument: 'config'
[2024-12-17 22:24:54,337: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-17 22:24:54,340: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 22:24:54,343: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 22:24:54,347: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 22:24:54,349: INFO: common: created directory at: artifacts]
[2024-12-17 22:24:54,351: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-17 22:24:54,352: INFO: data_ingestion: File already exists]
[2024-12-17 22:24:54,369: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-17 22:24:54,370: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-12-17 22:24:54,374: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 22:24:54,376: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 22:24:54,385: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 22:24:54,387: INFO: common: created directory at: artifacts]
[2024-12-17 22:24:54,389: INFO: common: created directory at: artifacts/data_validation]
[2024-12-17 22:24:54,455: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-12-17 22:24:54,456: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-12-17 22:24:54,460: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 22:24:54,462: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 22:24:54,469: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 22:24:54,470: INFO: common: created directory at: artifacts]
[2024-12-17 22:24:54,479: INFO: common: created directory at: artifacts/data_transformation]
[2024-12-17 22:24:54,548: INFO: data_transformation: Splitted data into training and test sets]
[2024-12-17 22:24:54,549: INFO: data_transformation: (1199, 12)]
[2024-12-17 22:24:54,550: INFO: data_transformation: (400, 12)]
[2024-12-17 22:24:54,551: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-12-17 22:54:08,533: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-17 22:54:08,539: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 22:54:08,544: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 22:54:08,552: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 22:54:08,556: INFO: common: created directory at: artifacts]
[2024-12-17 22:54:08,557: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-17 22:54:08,557: INFO: data_ingestion: File already exists]
[2024-12-17 22:54:08,579: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-17 22:54:08,585: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-12-17 22:54:08,593: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 22:54:08,595: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 22:54:08,655: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 22:54:08,674: INFO: common: created directory at: artifacts]
[2024-12-17 22:54:08,677: INFO: common: created directory at: artifacts/data_validation]
[2024-12-17 22:54:08,912: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-12-17 22:54:08,912: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-12-17 22:54:08,919: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 22:54:08,921: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 22:54:08,925: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 22:54:08,926: INFO: common: created directory at: artifacts]
[2024-12-17 22:54:08,940: INFO: common: created directory at: artifacts/data_transformation]
[2024-12-17 22:54:09,010: INFO: data_transformation: Splitted data into training and test sets]
[2024-12-17 22:54:09,011: INFO: data_transformation: (1199, 12)]
[2024-12-17 22:54:09,011: INFO: data_transformation: (400, 12)]
[2024-12-17 22:54:09,016: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-12-17 22:54:09,017: INFO: main: >>>>>> stage Model Training stage started <<<<<<]
[2024-12-17 22:54:09,023: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 22:54:09,026: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 22:54:09,036: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 22:54:09,038: INFO: common: created directory at: artifacts]
[2024-12-17 22:54:09,040: INFO: common: created directory at: artifacts/model_trainer]
[2024-12-17 22:54:09,041: ERROR: main: ModelTrainerConfig.__init__() got an unexpected keyword argument 'l1_ration']
Traceback (most recent call last):
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\main.py", line 46, in <module>
    model_trainer.initiate_model_training()
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\src\datascience\pipeline\model_trainer_pipeline.py", line 12, in initiate_model_training
    model_trainer_config = config.get_model_trainer_config()
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\src\datascience\config\configuration.py", line 66, in get_model_trainer_config
    model_trainer_config = ModelTrainerConfig(
TypeError: ModelTrainerConfig.__init__() got an unexpected keyword argument 'l1_ration'
[2024-12-17 22:55:02,076: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-17 22:55:02,082: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 22:55:02,085: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 22:55:02,091: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 22:55:02,097: INFO: common: created directory at: artifacts]
[2024-12-17 22:55:02,104: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-17 22:55:02,105: INFO: data_ingestion: File already exists]
[2024-12-17 22:55:02,120: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-17 22:55:02,121: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-12-17 22:55:02,128: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 22:55:02,130: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 22:55:02,135: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 22:55:02,138: INFO: common: created directory at: artifacts]
[2024-12-17 22:55:02,139: INFO: common: created directory at: artifacts/data_validation]
[2024-12-17 22:55:02,204: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-12-17 22:55:02,205: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-12-17 22:55:02,209: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 22:55:02,212: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 22:55:02,222: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 22:55:02,224: INFO: common: created directory at: artifacts]
[2024-12-17 22:55:02,225: INFO: common: created directory at: artifacts/data_transformation]
[2024-12-17 22:55:02,272: INFO: data_transformation: Splitted data into training and test sets]
[2024-12-17 22:55:02,272: INFO: data_transformation: (1199, 12)]
[2024-12-17 22:55:02,273: INFO: data_transformation: (400, 12)]
[2024-12-17 22:55:02,274: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-12-17 22:55:02,274: INFO: main: >>>>>> stage Model Training stage started <<<<<<]
[2024-12-17 22:55:02,278: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-17 22:55:02,280: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-17 22:55:02,284: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-17 22:55:02,289: INFO: common: created directory at: artifacts]
[2024-12-17 22:55:02,291: INFO: common: created directory at: artifacts/model_trainer]
[2024-12-17 22:55:02,310: INFO: main: >>>>>> stage Model Training stage completed <<<<<<

x==========x]
[2024-12-18 00:01:24,602: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-18 00:01:24,609: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:01:24,613: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:01:24,616: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:01:24,619: INFO: common: created directory at: artifacts]
[2024-12-18 00:01:24,622: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-18 00:01:24,623: INFO: data_ingestion: File already exists]
[2024-12-18 00:01:24,648: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-18 00:01:24,648: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-12-18 00:01:24,663: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:01:24,666: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:01:24,690: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:01:24,693: INFO: common: created directory at: artifacts]
[2024-12-18 00:01:24,694: INFO: common: created directory at: artifacts/data_validation]
[2024-12-18 00:01:24,775: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-12-18 00:01:24,776: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-12-18 00:01:24,780: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:01:24,784: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:01:24,791: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:01:24,793: INFO: common: created directory at: artifacts]
[2024-12-18 00:01:24,794: INFO: common: created directory at: artifacts/data_transformation]
[2024-12-18 00:01:24,848: INFO: data_transformation: Splitted data into training and test sets]
[2024-12-18 00:01:24,848: INFO: data_transformation: (1199, 12)]
[2024-12-18 00:01:24,849: INFO: data_transformation: (400, 12)]
[2024-12-18 00:01:24,851: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-12-18 00:01:24,853: INFO: main: >>>>>> stage Model Training stage started <<<<<<]
[2024-12-18 00:01:24,861: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:01:24,863: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:01:24,866: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:01:24,869: INFO: common: created directory at: artifacts]
[2024-12-18 00:01:24,872: INFO: common: created directory at: artifacts/model_trainer]
[2024-12-18 00:01:24,892: INFO: main: >>>>>> stage Model Training stage completed <<<<<<

x==========x]
[2024-12-18 00:04:32,696: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-18 00:04:32,707: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:04:32,710: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:04:32,717: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:04:32,723: INFO: common: created directory at: artifacts]
[2024-12-18 00:04:32,725: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-18 00:04:32,725: INFO: data_ingestion: File already exists]
[2024-12-18 00:04:32,740: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-18 00:04:32,741: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-12-18 00:04:32,754: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:04:32,757: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:04:32,761: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:04:32,762: INFO: common: created directory at: artifacts]
[2024-12-18 00:04:32,771: INFO: common: created directory at: artifacts/data_validation]
[2024-12-18 00:04:32,835: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-12-18 00:04:32,835: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-12-18 00:04:32,841: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:04:32,844: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:04:32,856: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:04:32,857: INFO: common: created directory at: artifacts]
[2024-12-18 00:04:32,858: INFO: common: created directory at: artifacts/data_transformation]
[2024-12-18 00:04:32,908: INFO: data_transformation: Splitted data into training and test sets]
[2024-12-18 00:04:32,909: INFO: data_transformation: (1199, 12)]
[2024-12-18 00:04:32,910: INFO: data_transformation: (400, 12)]
[2024-12-18 00:04:32,911: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-12-18 00:04:32,912: INFO: main: >>>>>> stage Model Training stage started <<<<<<]
[2024-12-18 00:04:32,922: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:04:32,924: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:04:32,928: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:04:32,930: INFO: common: created directory at: artifacts]
[2024-12-18 00:04:32,935: INFO: common: created directory at: artifacts/model_trainer]
[2024-12-18 00:04:32,957: INFO: main: >>>>>> stage Model Training stage completed <<<<<<

x==========x]
[2024-12-18 00:04:32,958: INFO: main: >>>>>> stage Model prediction stage started <<<<<<]
[2024-12-18 00:04:32,958: ERROR: main: [Errno 2] No such file or directory: 'artifacts\\model_trainer\\model.joblib']
Traceback (most recent call last):
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\main.py", line 56, in <module>
    prediction = PredictionPipeline()
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\src\datascience\pipeline\prediction_pipeline.py", line 8, in __init__
    self.model=joblib.load(Path("artifacts/model_trainer/model.joblib"))
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\venv\lib\site-packages\joblib\numpy_pickle.py", line 650, in load
    with open(filename, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts\\model_trainer\\model.joblib'
[2024-12-18 00:06:52,821: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-18 00:06:52,847: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:06:52,861: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:06:52,876: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:06:52,882: INFO: common: created directory at: artifacts]
[2024-12-18 00:06:52,895: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-18 00:06:52,898: INFO: data_ingestion: File already exists]
[2024-12-18 00:06:52,939: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-18 00:06:52,948: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-12-18 00:06:52,970: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:06:52,978: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:06:53,029: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:06:53,033: INFO: common: created directory at: artifacts]
[2024-12-18 00:06:53,037: INFO: common: created directory at: artifacts/data_validation]
[2024-12-18 00:06:53,129: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-12-18 00:06:53,131: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-12-18 00:06:53,146: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:06:53,150: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:06:53,159: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:06:53,165: INFO: common: created directory at: artifacts]
[2024-12-18 00:06:53,167: INFO: common: created directory at: artifacts/data_transformation]
[2024-12-18 00:06:53,243: INFO: data_transformation: Splitted data into training and test sets]
[2024-12-18 00:06:53,244: INFO: data_transformation: (1199, 12)]
[2024-12-18 00:06:53,245: INFO: data_transformation: (400, 12)]
[2024-12-18 00:06:53,247: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-12-18 00:06:53,248: INFO: main: >>>>>> stage Model Training stage started <<<<<<]
[2024-12-18 00:06:53,255: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:06:53,260: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:06:53,268: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:06:53,278: INFO: common: created directory at: artifacts]
[2024-12-18 00:06:53,283: INFO: common: created directory at: artifacts/model_trainer]
[2024-12-18 00:06:53,305: INFO: main: >>>>>> stage Model Training stage completed <<<<<<

x==========x]
[2024-12-18 00:06:53,311: INFO: main: >>>>>> stage Model prediction stage started <<<<<<]
[2024-12-18 00:06:53,316: ERROR: main: [Errno 2] No such file or directory: 'artifacts\\model_trainer\\model.joblib']
Traceback (most recent call last):
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\main.py", line 56, in <module>
    prediction = PredictionPipeline()
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\src\datascience\pipeline\prediction_pipeline.py", line 8, in __init__
    self.model=joblib.load(Path("artifacts/model_trainer/model.joblib"))
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\venv\lib\site-packages\joblib\numpy_pickle.py", line 650, in load
    with open(filename, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'artifacts\\model_trainer\\model.joblib'
[2024-12-18 00:20:10,862: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-18 00:20:10,869: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:20:10,871: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:20:10,875: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:20:10,877: INFO: common: created directory at: artifacts]
[2024-12-18 00:20:10,880: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-18 00:20:10,881: INFO: data_ingestion: File already exists]
[2024-12-18 00:20:10,898: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-18 00:20:10,899: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-12-18 00:20:10,908: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:20:10,913: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:20:10,919: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:20:10,925: INFO: common: created directory at: artifacts]
[2024-12-18 00:20:10,926: INFO: common: created directory at: artifacts/data_validation]
[2024-12-18 00:20:11,014: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-12-18 00:20:11,015: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-12-18 00:20:11,024: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:20:11,031: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:20:11,039: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:20:11,041: INFO: common: created directory at: artifacts]
[2024-12-18 00:20:11,043: INFO: common: created directory at: artifacts/data_transformation]
[2024-12-18 00:20:11,114: INFO: data_transformation: Splitted data into training and test sets]
[2024-12-18 00:20:11,116: INFO: data_transformation: (1199, 12)]
[2024-12-18 00:20:11,116: INFO: data_transformation: (400, 12)]
[2024-12-18 00:20:11,117: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-12-18 00:20:11,118: INFO: main: >>>>>> stage Model Training stage started <<<<<<]
[2024-12-18 00:20:11,119: ERROR: main: ModelTrainer.__init__() missing 1 required positional argument: 'config']
Traceback (most recent call last):
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\main.py", line 46, in <module>
    model_trainer = ModelTrainer()
TypeError: ModelTrainer.__init__() missing 1 required positional argument: 'config'
[2024-12-18 00:21:01,273: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-18 00:21:01,278: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:21:01,282: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:21:01,286: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:21:01,290: INFO: common: created directory at: artifacts]
[2024-12-18 00:21:01,292: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-18 00:21:01,294: INFO: data_ingestion: File already exists]
[2024-12-18 00:21:01,311: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-18 00:21:01,312: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-12-18 00:21:01,325: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:21:01,328: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:21:01,335: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:21:01,337: INFO: common: created directory at: artifacts]
[2024-12-18 00:21:01,338: INFO: common: created directory at: artifacts/data_validation]
[2024-12-18 00:21:01,398: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-12-18 00:21:01,402: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-12-18 00:21:01,407: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:21:01,409: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:21:01,412: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:21:01,416: INFO: common: created directory at: artifacts]
[2024-12-18 00:21:01,419: INFO: common: created directory at: artifacts/data_transformation]
[2024-12-18 00:21:01,468: INFO: data_transformation: Splitted data into training and test sets]
[2024-12-18 00:21:01,469: INFO: data_transformation: (1199, 12)]
[2024-12-18 00:21:01,470: INFO: data_transformation: (400, 12)]
[2024-12-18 00:21:01,470: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-12-18 00:21:01,471: INFO: main: >>>>>> stage Model Training stage started <<<<<<]
[2024-12-18 00:21:01,475: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:21:01,477: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:21:01,484: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:21:01,486: INFO: common: created directory at: artifacts]
[2024-12-18 00:21:01,487: INFO: common: created directory at: artifacts/model_trainer]
[2024-12-18 00:21:01,531: INFO: main: >>>>>> stage Model Training stage completed <<<<<<

x==========x]
[2024-12-18 00:21:01,532: INFO: main: >>>>>> stage Model prediction stage started <<<<<<]
[2024-12-18 00:21:01,548: ERROR: main: PredictionPipeline.predict() missing 1 required positional argument: 'data']
Traceback (most recent call last):
  File "C:\Users\steve\Desktop\Notebooks\datascienceproject\main.py", line 57, in <module>
    prediction.predict()
TypeError: PredictionPipeline.predict() missing 1 required positional argument: 'data'
[2024-12-18 00:34:11,494: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-18 00:34:11,504: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:34:11,508: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:34:11,520: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:34:11,523: INFO: common: created directory at: artifacts]
[2024-12-18 00:34:11,524: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-18 00:34:11,525: INFO: data_ingestion: File already exists]
[2024-12-18 00:34:11,545: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-18 00:34:11,547: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-12-18 00:34:11,562: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:34:11,564: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:34:11,574: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:34:11,574: INFO: common: created directory at: artifacts]
[2024-12-18 00:34:11,574: INFO: common: created directory at: artifacts/data_validation]
[2024-12-18 00:34:11,659: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-12-18 00:34:11,660: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-12-18 00:34:11,672: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:34:11,674: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:34:11,678: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:34:11,682: INFO: common: created directory at: artifacts]
[2024-12-18 00:34:11,684: INFO: common: created directory at: artifacts/data_transformation]
[2024-12-18 00:34:11,756: INFO: data_transformation: Splitted data into training and test sets]
[2024-12-18 00:34:11,756: INFO: data_transformation: (1199, 12)]
[2024-12-18 00:34:11,757: INFO: data_transformation: (400, 12)]
[2024-12-18 00:34:11,758: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-12-18 00:34:11,759: INFO: main: >>>>>> stage Model Training stage started <<<<<<]
[2024-12-18 00:34:11,764: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:34:11,767: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:34:11,772: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:34:11,775: INFO: common: created directory at: artifacts]
[2024-12-18 00:34:11,777: INFO: common: created directory at: artifacts/model_trainer]
[2024-12-18 00:34:11,827: INFO: main: >>>>>> stage Model Training stage completed <<<<<<

x==========x]
[2024-12-18 00:34:11,827: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<]
[2024-12-18 00:34:11,832: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:34:11,838: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:34:11,842: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:34:11,844: INFO: common: created directory at: artifacts]
[2024-12-18 00:34:11,846: INFO: common: created directory at: artifacts/model_evaluation]
[2024-12-18 00:34:18,576: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2024-12-18 00:34:40,947: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x]
[2024-12-18 00:47:31,683: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.8:8080]
[2024-12-18 00:47:31,686: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-12-18 00:47:39,800: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:47:39] "GET / HTTP/1.1" 200 -]
[2024-12-18 00:47:40,269: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:47:40] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2024-12-18 00:47:40,711: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:47:40] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-12-18 00:47:56,009: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:47:56] "POST /predict HTTP/1.1" 200 -]
[2024-12-18 00:48:26,788: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:48:26] "POST /predict HTTP/1.1" 200 -]
[2024-12-18 00:48:55,870: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.8:8080]
[2024-12-18 00:48:55,872: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-12-18 00:48:59,473: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:48:59] "GET / HTTP/1.1" 200 -]
[2024-12-18 00:49:00,018: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:49:00] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2024-12-18 00:49:11,991: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:49:11] "POST /predict HTTP/1.1" 200 -]
[2024-12-18 00:49:45,916: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.8:8080]
[2024-12-18 00:49:45,918: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-12-18 00:49:54,793: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:49:54] "GET / HTTP/1.1" 200 -]
[2024-12-18 00:49:55,311: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:49:55] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2024-12-18 00:50:11,488: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:50:11] "POST /predict HTTP/1.1" 200 -]
[2024-12-18 00:52:31,424: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.8:8080]
[2024-12-18 00:52:31,425: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-12-18 00:52:35,596: INFO: _internal: 127.0.0.1 - - [18/Dec/2024 00:52:35] "GET / HTTP/1.1" 200 -]
[2024-12-18 00:52:36,278: INFO: _internal: 127.0.0.1 - - [18/Dec/2024 00:52:36] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2024-12-18 00:52:36,482: INFO: _internal: 127.0.0.1 - - [18/Dec/2024 00:52:36] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -]
[2024-12-18 00:52:39,695: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:52:39] "GET / HTTP/1.1" 200 -]
[2024-12-18 00:52:40,284: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:52:40] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2024-12-18 00:53:29,534: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:53:29] "GET / HTTP/1.1" 200 -]
[2024-12-18 00:53:30,254: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:53:30] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2024-12-18 00:54:00,592: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-18 00:54:00,605: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:54:00,611: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:54:00,624: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:54:00,626: INFO: common: created directory at: artifacts]
[2024-12-18 00:54:00,628: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-18 00:54:00,632: INFO: data_ingestion: File already exists]
[2024-12-18 00:54:00,656: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-18 00:54:00,657: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-12-18 00:54:00,671: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:54:00,673: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:54:00,676: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:54:00,678: INFO: common: created directory at: artifacts]
[2024-12-18 00:54:00,683: INFO: common: created directory at: artifacts/data_validation]
[2024-12-18 00:54:00,778: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-12-18 00:54:00,779: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-12-18 00:54:00,792: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:54:00,795: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:54:00,812: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:54:00,818: INFO: common: created directory at: artifacts]
[2024-12-18 00:54:00,821: INFO: common: created directory at: artifacts/data_transformation]
[2024-12-18 00:54:00,929: INFO: data_transformation: Splitted data into training and test sets]
[2024-12-18 00:54:00,930: INFO: data_transformation: (1199, 12)]
[2024-12-18 00:54:00,932: INFO: data_transformation: (400, 12)]
[2024-12-18 00:54:00,950: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-12-18 00:54:00,955: INFO: main: >>>>>> stage Model Training stage started <<<<<<]
[2024-12-18 00:54:00,995: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:54:01,003: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:54:01,021: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:54:01,023: INFO: common: created directory at: artifacts]
[2024-12-18 00:54:01,026: INFO: common: created directory at: artifacts/model_trainer]
[2024-12-18 00:54:01,099: INFO: main: >>>>>> stage Model Training stage completed <<<<<<

x==========x]
[2024-12-18 00:54:01,101: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<]
[2024-12-18 00:54:01,106: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:54:01,108: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:54:01,119: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:54:01,120: INFO: common: created directory at: artifacts]
[2024-12-18 00:54:01,122: INFO: common: created directory at: artifacts/model_evaluation]
[2024-12-18 00:54:03,633: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2024-12-18 00:54:21,463: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:54:21] "GET / HTTP/1.1" 200 -]
[2024-12-18 00:54:21,586: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:54:21] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2024-12-18 00:54:30,566: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x]
[2024-12-18 00:54:31,159: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:54:31] "GET /train HTTP/1.1" 200 -]
[2024-12-18 00:54:38,808: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:54:38] "POST /predict HTTP/1.1" 200 -]
[2024-12-18 00:56:40,419: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-18 00:56:40,431: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:56:40,434: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:56:40,439: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:56:40,450: INFO: common: created directory at: artifacts]
[2024-12-18 00:56:40,451: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-18 00:56:40,452: INFO: data_ingestion: File already exists]
[2024-12-18 00:56:40,468: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-18 00:56:40,469: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-12-18 00:56:40,476: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:56:40,484: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:56:40,488: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:56:40,498: INFO: common: created directory at: artifacts]
[2024-12-18 00:56:40,499: INFO: common: created directory at: artifacts/data_validation]
[2024-12-18 00:56:40,571: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-12-18 00:56:40,572: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-12-18 00:56:40,581: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:56:40,583: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:56:40,588: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:56:40,592: INFO: common: created directory at: artifacts]
[2024-12-18 00:56:40,594: INFO: common: created directory at: artifacts/data_transformation]
[2024-12-18 00:56:40,648: INFO: data_transformation: Splitted data into training and test sets]
[2024-12-18 00:56:40,648: INFO: data_transformation: (1199, 12)]
[2024-12-18 00:56:40,649: INFO: data_transformation: (400, 12)]
[2024-12-18 00:56:40,650: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-12-18 00:56:40,651: INFO: main: >>>>>> stage Model Training stage started <<<<<<]
[2024-12-18 00:56:40,655: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:56:40,660: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:56:40,664: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:56:40,665: INFO: common: created directory at: artifacts]
[2024-12-18 00:56:40,667: INFO: common: created directory at: artifacts/model_trainer]
[2024-12-18 00:56:40,697: INFO: main: >>>>>> stage Model Training stage completed <<<<<<

x==========x]
[2024-12-18 00:56:40,701: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<]
[2024-12-18 00:56:40,706: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 00:56:40,715: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 00:56:40,719: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 00:56:40,722: INFO: common: created directory at: artifacts]
[2024-12-18 00:56:40,723: INFO: common: created directory at: artifacts/model_evaluation]
[2024-12-18 00:56:43,110: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2024-12-18 00:57:02,324: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x]
[2024-12-18 00:57:02,850: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 00:57:02] "GET /train HTTP/1.1" 200 -]
[2024-12-18 01:01:44,553: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 01:01:44] "GET /predict HTTP/1.1" 200 -]
[2024-12-18 01:01:44,666: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 01:01:44] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2024-12-18 01:02:02,339: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 01:02:02] "POST /predict HTTP/1.1" 200 -]
[2024-12-18 01:04:04,681: INFO: _internal: [31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on all addresses (0.0.0.0)
 * Running on http://127.0.0.1:8080
 * Running on http://192.168.1.8:8080]
[2024-12-18 01:04:04,683: INFO: _internal: [33mPress CTRL+C to quit[0m]
[2024-12-18 01:04:19,482: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 01:04:19] "GET / HTTP/1.1" 200 -]
[2024-12-18 01:04:20,136: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 01:04:20] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2024-12-18 01:04:30,754: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2024-12-18 01:04:30,769: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 01:04:30,778: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 01:04:30,783: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 01:04:30,785: INFO: common: created directory at: artifacts]
[2024-12-18 01:04:30,787: INFO: common: created directory at: artifacts/data_ingestion]
[2024-12-18 01:04:30,794: INFO: data_ingestion: File already exists]
[2024-12-18 01:04:30,813: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2024-12-18 01:04:30,813: INFO: main: >>>>>> stage Data Validation stage started <<<<<<]
[2024-12-18 01:04:30,820: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 01:04:30,827: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 01:04:30,832: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 01:04:30,834: INFO: common: created directory at: artifacts]
[2024-12-18 01:04:30,835: INFO: common: created directory at: artifacts/data_validation]
[2024-12-18 01:04:30,899: INFO: main: >>>>>> stage Data Validation stage completed <<<<<<

x==========x]
[2024-12-18 01:04:30,900: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2024-12-18 01:04:30,908: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 01:04:30,910: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 01:04:30,915: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 01:04:30,916: INFO: common: created directory at: artifacts]
[2024-12-18 01:04:30,918: INFO: common: created directory at: artifacts/data_transformation]
[2024-12-18 01:04:30,967: INFO: data_transformation: Splitted data into training and test sets]
[2024-12-18 01:04:30,967: INFO: data_transformation: (1199, 12)]
[2024-12-18 01:04:30,968: INFO: data_transformation: (400, 12)]
[2024-12-18 01:04:30,969: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<

x==========x]
[2024-12-18 01:04:30,970: INFO: main: >>>>>> stage Model Training stage started <<<<<<]
[2024-12-18 01:04:30,982: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 01:04:30,984: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 01:04:30,993: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 01:04:30,995: INFO: common: created directory at: artifacts]
[2024-12-18 01:04:30,996: INFO: common: created directory at: artifacts/model_trainer]
[2024-12-18 01:04:31,029: INFO: main: >>>>>> stage Model Training stage completed <<<<<<

x==========x]
[2024-12-18 01:04:31,030: INFO: main: >>>>>> stage Model evaluation stage started <<<<<<]
[2024-12-18 01:04:31,042: INFO: common: yaml file: config\config.yaml loaded successfully]
[2024-12-18 01:04:31,045: INFO: common: yaml file: params.yaml loaded successfully]
[2024-12-18 01:04:31,050: INFO: common: yaml file: schema.yaml loaded successfully]
[2024-12-18 01:04:31,054: INFO: common: created directory at: artifacts]
[2024-12-18 01:04:31,060: INFO: common: created directory at: artifacts/model_evaluation]
[2024-12-18 01:04:33,360: INFO: common: json file saved at: artifacts\model_evaluation\metrics.json]
[2024-12-18 01:04:52,818: INFO: main: >>>>>> stage Model evaluation stage completed <<<<<<

x==========x]
[2024-12-18 01:04:53,437: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 01:04:53] "GET /train HTTP/1.1" 200 -]
[2024-12-18 01:05:06,105: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 01:05:06] "GET /predict HTTP/1.1" 200 -]
[2024-12-18 01:05:06,231: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 01:05:06] "[33mGET /static/assets/img/form-v9.jpg HTTP/1.1[0m" 404 -]
[2024-12-18 01:05:27,890: INFO: _internal: 192.168.1.8 - - [18/Dec/2024 01:05:27] "POST /predict HTTP/1.1" 200 -]
